src/Microsoft.Identity.Client/Internal/MsalHelpers.cs

```diff
@@ -35,6 +35,16 @@ namespace Microsoft.Identity.Client.Internal
 {
     internal static class MsalHelpers
     {
+        public static string ToStringInvariant(this int val)
```

Comment:
nit: consider renaming to ToStringInvariantCulture or ToCultureInvariantString, the name seems to imply you are returning a nonMutable string